examples/03_connectivity/plot_seed_to_voxel_correlation.py

```diff
@@ -133,48 +133,55 @@
-                          seed_time_series.shape[0]
+seed_to_voxel_correlations = np.dot(brain_time_series.T, seed_time_series) / \
+                             seed_time_series.shape[0]
 
 ################################################
 # The resulting array will contain a value representing the correlation
 # values between the signal in the **seed region** of interest and **each
 # voxel's signal**, and will be of shape (n_voxels, 1). The correlation
 # values can potentially range between -1 and 1.
-print("seed-based correlation shape: (%s, %s)" % seed_based_correlations.shape)
-print("seed-based correlation: min = %.3f; max = %.3f" % (
-    seed_based_correlations.min(), seed_based_correlations.max()))
+print("Seed-to-voxel correlation shape: (%s, %s)" %
+      seed_to_voxel_correlations.shape)
+print("Seed-to-voxel correlation: min = %.3f; max = %.3f" % (
+    seed_to_voxel_correlations.min(), seed_to_voxel_correlations.max()))
+
+
+##########################################################################
+# Plotting the seed-to-voxel correlation map
+# ------------------------------------------
+# We can now plot the seed-to-voxel correlation map and perform thresholding
+# to only show values more extreme than +/- 0.5. Before displaying,
+# we need to create an in memory Nifti image object.
+# Furthermore, we can display the location of the seed with a sphere and
+# set the cross to the center of the seed region of interest.
+from nilearn import plotting
+seed_to_voxel_correlations_img = brain_masker.inverse_transform(
+    seed_to_voxel_correlations.T)
+display = plotting.plot_stat_map(seed_to_voxel_correlations_img,
+                                 threshold=0.5, vmax=1,
+                                 cut_coords=pcc_coords[0],
+                                 title=("Seed-to-voxel correlation "
+                                       "(PCC seed)"))
+display.add_markers(marker_coords=pcc_coords, marker_color='g',
+                    marker_size=300)
+# At last, we save the plot as pdf.
+display.savefig('pcc_seed_correlation.pdf')
 
 
 ##########################################################################
 # Fisher-z transformation and save nifti
 # --------------------------------------
-# Now we can Fisher-z transform the data to achieve a normal distribution.
+# Finally, we can Fisher-z transform the data to achieve a normal distribution.
 # The transformed array can now have values more extreme than +/- 1.
-seed_based_correlations_fisher_z = np.arctanh(seed_based_correlations)
-print("seed-based correlation Fisher-z transformed: min = %.3f; max = %.3f" % (
-    seed_based_correlations_fisher_z.min(),
-    seed_based_correlations_fisher_z.max()))
+seed_to_voxel_correlations_fisher_z = np.arctanh(seed_to_voxel_correlations)
+print("Seed-to-voxel correlation Fisher-z transformed: min = %.3f; max = %.3f"
+      % (seed_to_voxel_correlations_fisher_z.min(),
+         seed_to_voxel_correlations_fisher_z.max()))
```

Comment:
Same here. Placing the last 2 closing parentheses in separate lines will make this easier to parse, and will be PEP8 compliant.

Reply:
done